content/blog/posts/covid-19-data-scant-during-delta-variant-wave.md

```diff
@@ -0,0 +1,18 @@
+---
+title: COVID-19 data scant during Delta variant wave
+date: 2021-09-20
+authors:
+  - Eva Ruth Moravec
+hero: https://res.cloudinary.com/texas-justice-initiative/image/upload/v1632168497/5_ubwzqr.png
+---
+For months, Texas has been battling the highly contagious Delta strain of the coronavirus. After a surge in the beginning of 2021, [deaths due to COVID-19](https://apps.texastribune.org/features/2020/texas-coronavirus-cases-map/#new-deaths-anchor) were somewhat low in the spring – but then they started to climb. 
+
+And despite the rise in cases undoubtedly affecting individuals incarcerated in prisons and jails, who are particularly vulnerable to die from COVID-19, the actual reporting of infections and deaths in Texas lockups has mostly stopped. 
+
+For many months since the start of the pandemic, the Texas Commission on Jail Standards, which is responsible for ensuring county jails operate within state law, had asked jails to file [daily reports](https://www.tcjs.state.tx.us/wp-content/uploads/2020/12/TA_Memo-COVID_Reporting.pdf) on whether (and how many) infections were detected among staff and incarcerated individuals. However, that practice ended in June, when TCJS informed jails that the reports were no longer required. 
+
+Throughout the pandemic, the Texas Department of Criminal Justice, which oversees prisons, periodically announced deaths of employees and incarcerated individuals [online](https://www.tdcj.texas.gov/covid-19/index2.html). Yet no deaths of incarcerated individuals have been announced since mid-June. 
+
+Amid this dearth of data on COVID-19 deaths in Texas lockups, since mid-June, more than 9,500 Texans have died from the coronavirus, according to state data. Also since mid-June, the pandemic has killed at least [23 people who worked](https://texasjusticeinitiative.org/publications/covid-law-enforcement-deaths) for county sheriff's offices and TDCJ, according to media sources, agencies and the [Officer Down Memorial Page](https://www.odmp.org/search?cause=COVID19&state=texas&o=). 
+
+So just how many people have died of COVID-19 in Texas prisons and jails during this latest deadly wave? [At least eight](https://texasjusticeinitiative.org/publications/covid-deaths-in-texas) – according to custodial death reports that agencies are required by state law to file within 30 days of a death. But the true number? It will take months to know for sure. 
```

Comment:
```suggestion
So just how many people have died of COVID-19 in Texas prisons and jails during this latest deadly wave? [At least 11](https://texasjusticeinitiative.org/publications/covid-deaths-in-texas) – according to custodial death reports that agencies are required by state law to file within 30 days of a death. But the true number? It will take months to know for sure. 
```